### PR TITLE
Deleted revisions samples

### PR DESCRIPTION
**Pull request by MinervaLong**
Hello @brendajerop I'm not sure if it's too late but ¿Could you check my pull request? :) 
Here is the link to my Sandbox:
https://www.mediawiki.org/wiki/User:VanesaPerea/Sandbox/API:Deletedrevisions

Thank you

### CODE REVIEW

**Comment by MinervaLong** [comment]
Hello @brendajerop thank you for your feedback, I've tried to complete your suggestions

File: javascript/get_deleted_revs.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+// Step 3: GET request to get a list of deleted revisions
+
+var url = "https://en.wikipedia.org/w/api.php"; 
+
+var params_2 = {
+    action: "query",
+    format: "json",
+    titles: "Talk:MainPage",
+    prop:   "deletedrevisions",
+    drv:    "prop"
```
Comment (brendajerop) [inline javascript/get_deleted_revs.js]: @MinervaLong I don't think drv is a a parameter of this module. Maybe you meant, `drvprop`?

File: javascript/get_deleted_revs.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/*
+    get_deleted_revs.js
+
+    MediaWiki API Demos
+    Demo of `Deleted revisions:Get a list of deleted revision for Talk:Main Page` module
```
Comment (brendajerop) [inline javascript/get_deleted_revs.js]: Let's make this "Demo of `Deleted Revisions` module: Get a list of deleted revisions for a page" . Remove the page specification in your README files as well, so as to make the module description clearer :)

File: python/get_deleted_revs.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+R = S.get(url=URL, params=PARAMS_0)
+DATA = R.json()
+
+LOGIN_TOKEN = DATA['query']['tokens']['logintoken']
+
+# Send a post request to login. Using the main account for login is not
+# supported. Obtain credentials via Special:BotPasswords
+# (https://www.mediawiki.org/wiki/Special:BotPasswords) for lgname & lgpassword
+
```
Comment (brendajerop) [inline python/get_deleted_revs.py]: Remove this blank line

File: python/get_deleted_revs.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+import requests
+
+S = requests.Session()
+
+URL = "https://en.wikipedia.org/w/api.php"
+
+# Retrieve login token
+
```
Comment (brendajerop) [inline python/get_deleted_revs.py]: Remove this blank line.

File: python/get_deleted_revs.py [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    'lgname':"your_bot_username",
+    'lgpassword':"your_bot_password",
+    'lgtoken':LOGIN_TOKEN,
+    'format':"json"
+}
+
+R = S.post(URL, data=PARAMS_1)
+
+#Send a get request to get the list of deleted revisions
+
```
Comment (brendajerop) [inline python/get_deleted_revs.py]: Remove the two blank lines.

**Comment by MinervaLong** [comment]
I'll work on this tis week @brendajerop  thank you

File: javascript/get_deleted_revs.js (hunk 1 above)
Comment (MinervaLong) [reply javascript/get_deleted_revs.js]: @brendajerop I've installed lint checkers in VSC and I fix some mistakes it indicates and I've revised the template and change for drprop parameter

File: javascript/get_deleted_revs.js (hunk 2 above)
Comment (MinervaLong) [reply javascript/get_deleted_revs.js]: I've done it in the others as well, hope it was right :)

**Pull request merged**